pkg/security/podsecuritypolicy/group/types.go

```diff
@@ -31,5 +31,5 @@ type GroupStrategy interface {
 	// value to return if configured with multiple ranges.  This is used for FSGroup.
 	GenerateSingle(pod *api.Pod) (*int64, error)
 	// Validate ensures that the specified values fall within the range of the strategy.
-	Validate(fldPath *field.Path, pod *api.Pod, groups []int64) field.ErrorList
+	Validate(fldPath *field.Path, pod *api.Pod, runAsNonRoot *bool, groups []int64) field.ErrorList
```

Comment:
defer the runAsNonRoot aspects until https://github.com/kubernetes/kubernetes/pull/62216 is settled?